py/parse_and_notify.py

```diff
@@ -145,53 +149,50 @@ def build(cls, raw_content: Mapping[str, Any]) -> UpdatedTopic:
 @dataclass(frozen=True)
 class TopicctlOutput:
     dry_run: bool
-    topics: Sequence[Topic]
+    topic: Topic
```

Comment:
This class has no meaning anymore.
Please remove it. Move the `build` method in the `Topic` class https://github.com/getsentry/topicctl/pull/14/files#diff-ef63568addaf1ff2f63d1d162f6bf246aacd94fb0a741dfd5e3bcc39df440e14R40-R45. Let's not leave a design that does not represent the data mdoel anymore around.